.github/workflows/govern-ci-cd.yml

```diff
@@ -0,0 +1,35 @@
+name: Govern API tests
```

Comment:
Can you elaborate closer here or in the PR desc why it was required to add this workflow within this scope?

Reply:
Perf. thanks.. and why ever this got removed 🤷